virtcontainers/fc.go

```diff
@@ -510,6 +666,36 @@ func (fc *firecracker) createDiskPool() error {
 	return nil
 }
 
+func (fc *firecracker) umountResource(jailedPath string) {
+	hostPath := filepath.Join(fc.jailerRoot, jailedPath)
+	syscall.Unmount(hostPath, syscall.MNT_DETACH)
```

Comment:
The lazy unmount could still fail if the path is incorrect somehow, so I'd log the return value or return it anyway.

Reply:
done